db/md/DATABASE-SETUP.md

```
@@ -0,0 +1,91 @@
+# FPL Live Database Setup Guide
+
+## Quick Setup Steps
+
+### 1. Run Database Migration
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)